subprojects/base-services/src/main/java/org/gradle/internal/operations/BuildOperation.java

```diff
@@ -16,12 +16,14 @@
 
 package org.gradle.internal.operations;
 
+import org.gradle.internal.progress.BuildOperationDescriptor;
+
 /**
  * Represents some chunk of work.
  */
 public interface BuildOperation {
     /**
      * Returns a human consumable name for this operation.
      */
-    String getDescription();
+    BuildOperationDescriptor.Builder description();
```

Comment:
This should line up with `Describable`. We should continue to evolve a common concept of things that have display names.